src/rpm/package.rs

```diff
@@ -699,23 +701,23 @@ impl RPMPackageMetadata {
     ///
     /// Note that this is not necessarily the same as the digest
     /// used for headers.
-    pub fn get_file_digest_algorithm(&self) -> Result<FileDigestAlgorithm, RPMError> {
+    pub fn get_file_digest_algorithm(&self) -> Result<DigestAlgorithm, RPMError> {
         self.header
             .get_entry_data_as_u32(IndexTag::RPMTAG_FILEDIGESTALGO)
             .and_then(|x| {
-                FileDigestAlgorithm::from_u32(x).ok_or_else(|| {
-                    RPMError::InvalidTagValueEnumVariant {
-                        tag: IndexTag::RPMTAG_FILEDIGESTALGO.to_string(),
-                        variant: x,
-                    }
+                DigestAlgorithm::from_u32(x).ok_or_else(|| RPMError::InvalidTagValueEnumVariant {
+                    tag: IndexTag::RPMTAG_FILEDIGESTALGO.to_string(),
+                    variant: x,
                 })
             })
     }
 
     /// Extract a the set of contained file names including the additional metadata.
     pub fn get_file_entries(&self) -> Result<Vec<FileEntry>, RPMError> {
         // rpm does not encode it, if it is the default md5
-        let algorithm = self.get_file_digest_algorithm().unwrap_or_default();
+        let algorithm = self
+            .get_file_digest_algorithm()
+            .unwrap_or(DigestAlgorithm::Md5);
```

Comment:
Change: deleted `Default` impl for DigestAlgorithm as it should probably be contextual

Change: renamed a few fields named after `sense` to `flags`